### ISSUE TITLE
Unable to visually determine selected tab in file details view when using keyboard navigation

### ISSUE THREAD

**Issue by kevgathuku** [nextcloud/server#9944]
### Steps to reproduce
1. Open the files view.  For any file, open the file details view
2. Try to navigate the tabs (Activities, Comments, Sharing, Versions) using the keyboard
3. Nothing visible is happening. It is unclear which element currently has focus

### Expected behaviour
The tab element in focus should have a visible outline around it to enable the user to visually determine the component on which keyboard operations will interact at any point in time.

<img width="351" alt="screen shot 2018-06-21 at 12 01 29" src="https://user-images.githubusercontent.com/3721994/41709272-7680987a-754b-11e8-932a-ebd83595c938.png">
☝️ expected output when the sharing tab is focused with the keyboard

### Actual behaviour
There is no visual indicator of the element that currently has keyboard focus
<img width="350" alt="screen shot 2018-06-21 at 12 00 57" src="https://user-images.githubusercontent.com/3721994/41709376-b247cdd8-754b-11e8-9cbe-a6356b41af06.png">

For this image, when a user interacts with the keyboard, it is not possible to tell which element they would be interacting with.

### Server configuration

**Operating system**: Ubuntu 16.04

**Nextcloud version:** 13.0.4

**Updated from an older Nextcloud/ownCloud or fresh install:** Fresh install

**Where did you install Nextcloud from:** `snap install nextcloud`

**Comment by MorrisJobke** [comment]
cc @nextcloud/designers

**Comment by kevgathuku** [comment]
Thanks @jancborchardt 
I'll work on a fix, and I'll make sure to ask any questions that may come up 😃

**Comment by kevgathuku** [comment]
Fixed in #9956

**Comment by jancborchardt** [comment]
It’s awesome when the person who finds the issue also fixes it! :) High five @kevgathuku
